src/components/FeatureFlags/FeatureFlagsProvider.tsx

```diff
@@ -22,8 +23,9 @@ const config: IFlagProvider['config'] = {
       .fetch(url, headers)
       .then((resp) => {
         // prevent the request from falling back to default error behavior
```

Comment:
Correct me if I'm wrong, but doesn't this change "re-enable" bubbling this error up to the default error handler?

Reply:
No, there is a catch branch below, that is the actual error handler.

Reply:
I see, thanks!